README.rst

```diff
@@ -88,7 +88,7 @@ You can run tests by executing::
 
     virtualenv env
     source env/bin/activate
-    pip install -r tests/requirements.txt
+    pip install -r test_requirements/base.txt
```

Comment:
I'd prefer to have the test requirements inside the tests/ folder to keep it clean, like tests/requirements/base.txt, let me know if that would break best practices.